test/acl_usm_test.cpp

```diff
@@ -1206,4 +1244,74 @@ MT_TEST(acl_usm, memfill_usm) {
   ACL_LOCKED(acl_print_debug_msg("end memfill_usm\n"));
 }
 
+MT_TEST(acl_usm, read_device_global) {
+  ACL_LOCKED(acl_print_debug_msg("begin read_device_global\n"));
+  char str[100];
+  const size_t strsize = sizeof(str) / sizeof(char); // includes NUL (!)
+  char resultbuf[strsize];
+  cl_int status;
+
+  cl_event write_event = 0;
+  cl_event read_event = 0;
+
+  // Prepare host memory
+  syncThreads();
+  // Host pointer example
+  void *src_ptr = malloc(strsize);
+  CHECK(src_ptr != NULL);
+  // Host USM example
+  // void* src_ptr = clHostMemAllocINTEL(m_context, NULL, strsize, 1, &status);
+  // CHECK_EQUAL(CL_SUCCESS, status);
+  // CHECK(src_ptr != NULL);
```

Comment:
Commented code should be removed; unless you are intend to enable this before merge?